src/style.css

```diff
@@ -1 +1,64 @@
-/* Styles go here */
+:root {
+  --hover-color: #00acdc;
+}
+
+body {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.header {
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  margin: 10px 50px;
+}
+
+.container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav {
+  padding-bottom: 6px;
+  padding-right: 34px;
+}
+
+.nav-item {
+  display: inline-block;
+  margin-right: 16px;
```

Comment:
Due to mockup this indent should be 20px instead of 16px
![image](https://user-images.githubusercontent.com/80700247/233826654-5e3a2a07-ef61-4678-a3c7-302d9595d249.png)